syntax = "proto3";

package trustlevel;

message Input {
    string input_string = 1;
}

message BiasOutput {
    double score = 1;
    repeated string explanations = 2;
}

service ServiceDefinition {
    rpc determineBias(Input) returns (BiasOutput) {}
}
